front_end/src/app/(main)/questions/[id]/components/forecast_maker/forecast_maker_question/forecast_maker_continuous.tsx

```diff
@@ -68,7 +72,11 @@ const ForecastMakerContinuous: FC<Props> = ({
   const [weights, setWeights] = useState<number[]>(
     prevForecastValue?.weights ?? [1]
   );
-  const previousForecast = question.my_forecasts?.latest;
+  const previousForecast =
+    !!question.my_forecasts?.latest &&
+    isNil(question.my_forecasts.latest.end_time)
+      ? question.my_forecasts.latest
+      : undefined;
```

Comment:
Similar note as for previous forecast makers. We have prev forecast pasted through props, extracted prev forecast value (previously BE returned `any` object, so this was done to ensure we're working with valid data on FE) and `previousForecast`. I think this should be normalized to have a single source for previous forecast

Reply:
That was specifically for slider values.
the issue with using slider values is that not all continuous forecasts have slider values. And in the future, we would want the storage of the "input method" to be more options than just slider values.
Unfortunately, my sidestepping of using the previously used `prevForecast` isn't complete, and probably should have been done as a separate PR entirely, but alas, I'm in quite deep in this PR, so I'll just have to integrate abandoning the method of passing around slider values in place of the full record of the lastest user (or aggregation) forecast.